src/assets/style/components/navbar.scss

```diff
@@ -0,0 +1,44 @@
+//Mixins
+@mixin textLinks {
+  color: $white-color;
+}
+
+@mixin funcionality($rogue) {
+  display: $rogue;
+}
+
+//Navbar
+.navbar {
+  @include navbarDefaultGroup;
+
+  &__right {
+    @include textLinks;
+  }
+
+  &__left {
+    &-img-responsive {
+      @include funcionality(none);
+    }
+  }
+}
+
+//Media Query
+@media (max-width: 450px) {
```

Comment:
This can be improved

delete __right since it is not necessary because don't change from original code